angularjs/manageVariable/edit.controller.js

```diff
@@ -265,6 +265,8 @@
                 return;
             }
 
+            this.variable.name = encodeURIComponent(this.variable.name);
```

Comment:
That works, but makes the UI showing the encoded value while the request is running. Might be better to make a local copy of the variable and only manipulate the value there to avoid this

Reply:
yes, true. I'll add that.